src/structure/utils.py

```diff
@@ -0,0 +1,385 @@
+# utils.py
+from __future__ import annotations
+import math
+from collections import defaultdict, namedtuple
+from typing import Dict, List, Tuple
+
+import numpy as np
+import biotite.structure as struc
+
+#________________HYDROGEN BONDS__________________________
+
+DA_MAX = 3.5           # Å donor–acceptor distance
```

Comment:
I think for now it's fine to leave all of these as globals. It'll be super easy to change this to something modifiable in the config file. There isn't much that's actually hard coded.